site/content/overview/gateway-architecture.md

```diff
@@ -21,7 +21,7 @@ NGINX Gateway Fabric is an open source project that provides an implementation o
 
 For a list of supported Gateway API resources and features, see the [Gateway API Compatibility]({{< relref "/overview/gateway-api-compatibility.md" >}}) documentation.
 
-We have more information regarding our [design principles](https://github.com/nginxinc/nginx-gateway-fabric/blob/v1.4.0/docs/developer/design-principles.md) in the project's GitHub repository.
+We have more information regarding our [design principles](https://github.com/nginxinc/nginx-gateway-fabric/blob/v1.5.0/docs/developer/design-principles.md) in the project's GitHub repository.
 
 ## NGINX Gateway Fabric at a high level
```

Comment:
@sjberman do we need to update this doc after the init container changes?

Reply:
I looked at it but I didn't think it really needed an update. Feels like this doc is more in depth about the architecture, and the init container just copies a couple files.